app/src/main/java/com/github/swent/echo/compose/authentication/createProfile.kt

```diff
@@ -0,0 +1,251 @@
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.InputChip
+import androidx.compose.material3.InputChipDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.SmallFloatingActionButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.DpOffset
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.unit.toSize
+import androidx.compose.ui.window.PopupProperties
+import androidx.navigation.compose.rememberNavController
+import com.github.swent.echo.ExcludeFromJacocoGeneratedReport
+import com.github.swent.echo.R
+import com.github.swent.echo.data.model.Section
+import com.github.swent.echo.data.model.SectionEPFL
+import com.github.swent.echo.data.model.Semester
+import com.github.swent.echo.data.model.SemesterEPFL
+import com.github.swent.echo.data.model.Tag
+import com.github.swent.echo.ui.navigation.NavigationActions
+
+/**
+ * A composable function that displays the UI for creating a user profile.
+ *
+ * @param sectionList The list of sections to be displayed in the dropdown menu.
+ * @param semList The list of semesters to be displayed in the dropdown menu.
+ * @param tagList The list of tags to be displayed as chips.
+ */
+@Composable
+fun ProfileCreationUI(
+    sectionList: List<Section>,
```

Comment:
I think those List are temporary, they will be fetched from the repository later. Instead of asking them as arguments for the ProfileCreationUI, you could add samples outside the function and call them when needed

Reply:
We actually already have those values in the code. Just use `SectionEPFL.values()` (same for `SemesterEPFL`).